tools/src/main/scala/org/apache/spark/sql/rapids/tool/AppFilterImpl.scala

```diff
@@ -18,16 +18,16 @@ package org.apache.spark.sql.rapids.tool
 
 import java.util.Calendar
 import java.util.concurrent.{ConcurrentLinkedQueue, Executors, ThreadPoolExecutor, TimeUnit}
-
 import scala.collection.JavaConverters._
-
 import com.google.common.util.concurrent.ThreadFactoryBuilder
 import com.nvidia.spark.rapids.tool.EventLogInfo
+import com.nvidia.spark.rapids.tool.EventLogPathProcessor.logError
 import com.nvidia.spark.rapids.tool.qualification.QualificationArgs
 import org.apache.hadoop.conf.Configuration
-
 import org.apache.spark.internal.Logging
 
+import scala.collection.mutable.LinkedHashMap
```

Comment:
this should go above with other scala imports

Reply:
done